src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/DocumentForm.js

```diff
@@ -13,24 +13,25 @@ import { BackOfficeRoutes } from '@routes/urls';
 import { getIn } from 'formik';
 import _get from 'lodash/get';
 import PropTypes from 'prop-types';
+import Overridable from 'react-overridable';
 import React, { Component } from 'react';
 import {
   AlternativeAbstracts,
   AlternativeIdentifiers,
   AlternativeTitles,
   AuthorsField,
+  ConferenceInfoField,
   Copyrights,
   Identifiers,
+  Imprint,
+  InternalNotes,
+  Keywords,
```

Comment:
Watch out for this `index.js` that will not trigger a circular dependency

Reply:
```
Processed 998 files (6.5s) (281 warnings)

✔ No circular dependency found!
```